scripts/test.sh

```diff
@@ -265,7 +265,7 @@ case "$TARGET" in
   ;;
 "eslint-ci")
   [[ $CONFIGURATION == 'Debug' ]] && exit 0
-  npm ci
+  npm ci --ignore-scripts
```

Comment:
why are we ignoring scripts in this change. Is this for security concerns?

Reply:
Because there seem to be an issue running `lerna bootstrap` off a `postinstall` script when running `npm ci` instead of `npm install` and this step doesn't need the realm binary to be built anyway.

Reply:
I reverted this change now - and instead we have to run `npx lerna bootstrap` as developers when checking out initially or switching branch.